docs/modules/operation/pages/provisioning/detectors/FtpDetector.adoc

```diff
@@ -0,0 +1,31 @@
+= FTP Service Detector
+
+The FtpDetector is able to validate ftp connection dial-up processes.
+The detector can test ftp server on multiple ports and specific login data.
+
+== Detector facts
+
+[options="autowidth"]
+|===
+| Implementation | `org.opennms.netmgt.provision.detector.simple.FtpDetector`
+|===
+
+== Configuration and Use
+
+.Parameters for the FTP Service Detector
+[options="header, %autowidth"]
+|===
+| Parameter            | Description   | Required | Default value
+| `port`               | Port to query | optional | `21`
+| `multilineIndicator` | ???           | optional | `-`
+|===
+
+== Example Configuration
+
+[source,xml]
+----
+<detector name="FTP" class="org.opennms.netmgt.provision.detector.simple.FtpDetector">
+  <parameter key="port" value="21"/>
+  <parameter key="multilineIndicator" value="-"/>
```

Comment:
Maybe adding an example with credentials?

Reply:
I thought this is not possible due to the missing parameters here:
![image](https://user-images.githubusercontent.com/4995222/120555152-eec41400-c3fa-11eb-8e03-0b1bee70be97.png)

Reply:
```suggestion
```